go.mod

```diff
@@ -0,0 +1,29 @@
+module github.com/mongodb/mongo-tools
+
+go 1.14
```

Comment:
The README.md says `go 1.15`, but this says `go 1.14`. Which one is correct?

Reply:
It's `go 1.15`, thanks for catching that. I thought I was using that version locally but I have `go 1.14`, which is why the go.mod file defaulted to that.